src/sonic-config-engine/minigraph.py

```diff
@@ -1458,6 +1458,33 @@ def select_mmu_profiles(profile, platform, hwsku):
                 base_file = os.path.join(path, file_item)
                 exec_cmd(["sudo", "cp", file_in_dir, base_file])
 
+def is_ipv4(address):
+    # encode and decode to unicode, because when address is bytes type, ip_network will throw AddressValueError
+    # set strict to False because address may set host bit, for example 192.168.0.1/24
+    return isinstance(ipaddress.ip_network(UNICODE_TYPE(address), False), ipaddress.IPv4Network)
+
+def update_forced_mgmt_route(mgmt_intf, mgmt_routes):
+    for mgmt_intf_key in mgmt_intf.keys():
+        forced_mgmt_routes = []
+
+        try:
+            # get mgmt interface type
+            mgmt_intf_addr = mgmt_intf_key[1]
+            mgmt_is_ipv4 = is_ipv4(mgmt_intf_addr)
+
+            # add mgmt route to different mgmt interface by address type
+            for mgmt_route in mgmt_routes:
+                route_is_ipv4 = is_ipv4(mgmt_route)
```

Comment:
change to route_iftype

Reply:
Agree type comparison is more readable than boolean comparison.

Reply:
Fixed, change to compare iftype.